tests/test_plotting.py

```diff
@@ -1,6 +1,8 @@
 import io
 from typing import Any
 
+from matplotlib.axes import Axes
+from matplotlib.figure import Figure
```

Comment:
I assume this just hides the explicit re-export issue (I'm surprised that mypy/pyright are fooled that easily).

Reply:
No, I think `matplotlib` got updated, and now `plt.Axes` and `plt.Figure` aren't valid any more in code from a type checking perspective.